tests/ui_tools/test_buttons.py

```diff
@@ -297,6 +303,7 @@ def test_keypress_emoji_button(
             controller=controller,
             emoji_unit=("+1", "1f44d", ["thumbs_up", "like"]),
             message=message_fixture,
+            reaction_count=1,
```

Comment:
Do we know enough here to assert on the reaction count after?

Reply:
We can't assert on the reaction_count currently, because of the below lambda functions. I can instead add `case`'es to the test in the previous commit and then assert on the reaction_count in this one, but that won't be "just a" keypress check anymore?

Reply:
I've added tests for checking reaction_count as well.  Let me know if that looks complex. :)